docs/diagnostics/PX1062.md

```diff
@@ -9,7 +9,7 @@ This document describes the PX1062 diagnostic.
 
 ## Diagnostic Description
 
-Static members in classes of the PXGraph type or its successors are strictly prohibited unless these members are read-only. The diagnostic shows the error for static properties and static fields such as views and actions.
+In graphs and graph extensions, static actions, views, and mutable fields and properties are strictly forbidden unless these members are read-only. The diagnostic shows the error for such static members.
```

Comment:
`unless these members are read-only` is not applicable to static views and actions but is read as if we allow static read-only views and actions. The word `mutable` is enough, it is antonym for the word `read-only`

```suggestion
In graphs and graph extensions, static actions, views, and mutable fields and properties are strictly forbidden. The diagnostic shows the error for such static members.
```